app/views/articles/_single_story.html.erb

```diff
@@ -4,11 +4,13 @@
     <div id="featured-story-marker" data-featured-article="articles-<%= story.id %>"></div>
   <% end %>
   <% if featured == true || (feed_style_preference == "rich" && story.main_image.present?) %>
-    <a href="<%= story.path %>" title="<%= story.title %>" aria-label="article"
-      style="background-color: <%= story.main_image_background_hex_color %>; background-image: url(<%= cloud_cover_url(story.main_image) %>);"
-      class="crayons-story__cover crayons-story__cover__image">
-      <span class="hidden"><%= story.title %></span>
-    </a>
+    <div class="crayons-article__cover crayons-article__cover__image__feed">
+      <a href="<%= story.path %>" title="<%= story.title %>" aria-label="article"
```

Comment:
```suggestion
      <a href="<%= story.path %>" title="<%= story.title %>"
```
Can we sneak in this quick a11y fix maybe? It's unrelated to your changes but it's stopping your image alt from having an impact on the overall link accessible name (and burning my eyes 😂 )